### PR TITLE
Mark system, owner and description options required.

### PR DESCRIPTION
**Pull request by ext-jaret**
All these options shall be provided for API auto-registration to work. Providing sample text as default value.

Co-authored-by: Dmitry Belyaev <dbelyaev@vmware.com>

### CODE REVIEW

File: java-rest-service/accelerator.yaml [hunk 1]
```diff
@@ -223,12 +219,19 @@ engine:
         chain:
           - type: RewritePath
             rewriteTo: "'config/workload.yaml'"
+    # End OpenAPI Endpoint Auto-registration
+
+    # Workload Resource name
+    - merge:
+      - exclude: [ "config/workload.yaml", "config/test-pipeline.yaml" ]
       - include: [ "config/workload.yaml", "config/test-pipeline.yaml" ]
```
Comment (trisberg) [inline java-rest-service/accelerator.yaml:227]: why exclude and include the same set of files?

Comment (trisberg) [reply java-rest-service/accelerator.yaml:227]: maybe add `     - include: [ "**" ]` before the exclude to make it more obvious

Comment (trisberg) [reply java-rest-service/accelerator.yaml:227]: and that is also the pattern used in "# OpenAPI Endpoint Auto-registration"

File: java-rest-service/accelerator.yaml [hunk 2]
```diff
@@ -57,22 +52,22 @@ accelerator:
     - name: apiSystem
       label: System API Belongs To
       inputType: text
-      required: false
-      defaultValue: ''
+      required: true
+      defaultValue: "Profile Management"
```
Comment (trisberg) [inline java-rest-service/accelerator.yaml:56]: system names that I have seen are lowercase and kebob-case - see https://backstage.io/docs/features/software-catalog/descriptor-format#kind-system

File: java-rest-service/accelerator.yaml [hunk 3]
```diff
@@ -57,22 +52,22 @@ accelerator:
...
       dependsOn:
         name: exposeOpenAPIEndpoint
     - name: apiOwner
       label: Owner of API
       inputType: text
-      required: false
-      defaultValue: ''
+      required: true
+      defaultValue: "Customer Relations Department"
```
Comment (trisberg) [inline java-rest-service/accelerator.yaml:63]: owners would be user or group I assume and they are typically lower case or kebab-case

File: java-rest-service/accelerator.yaml (hunk 1 above)
Comment (vmware-jaret) [reply java-rest-service/accelerator.yaml:227]: Merge require us to exclude the files we would like to change in order to keep all files.

File: java-rest-service/accelerator.yaml (hunk 2 above)
Comment (dmbelyaev) [reply java-rest-service/accelerator.yaml]: Done

File: java-rest-service/accelerator.yaml (hunk 3 above)
Comment (dmbelyaev) [reply java-rest-service/accelerator.yaml]: Done

**Review by trisberg — APPROVED** [review]

**Pull request merged**